agents/gce/fence_gce.py

```diff
@@ -20,6 +28,8 @@
 
 import googleapiclient.discovery
 from fencing import fail_usage, run_delay, all_opt, atexit_handler, check_input, process_input, show_docs, fence_action
+from oauth2client.client import GoogleCredentials
```

Comment:
Can do do these in a try/except: pass ?

That should make it able to be built (e.g. manpage generation) even on FreeBSD, where we dont have oauth2client currently installed.

Reply:
Added the try/except:pass to the oauth2client.* imports